app/app.go

```diff
@@ -14,11 +15,29 @@ import (
 // Set with the go linker flag -X.
 var extraArgs string
 
+// ID is the app id exposed to the platform.
+//
+// On Android ID is the package property of AndroidManifest.xml,
+// on iOS ID is the CFBundleIdentifier of the app Info.plist,
+// on Wayland it is the toplevel app_id,
+// on X11 it is the X11 XClassHint
+//
+// ID is set by the gogio tool or manually with the -X linker flag:
+//
+// EXAMPLE: gogio -ldflags="-X 'gioui.org/app.ID=org.gioui.example.Kitchen'" .
```

Comment:
I'd expected `go` in the example, such as:

```
...
ID is set by the gogio tool or manually with the -X linker flag. For example,

    go build -ldflags="-X 'gioui.org/app.ID=org.gioui.example.Kitchen'" .

```

I left out the concrete command so you could test it before we submit this PR.